src/api.rs

```diff
@@ -11,7 +11,7 @@ pub const CRYPTO_PUBLICKEYBYTES: usize = 261120;
 pub const CRYPTO_SECRETKEYBYTES: usize = 6492;
 #[cfg(any(feature = "mceliece348864", feature = "mceliece348864f"))]
 /// The number of bytes required to store the ciphertext resulting from the encryption
-pub const CRYPTO_CIPHERTEXTBYTES: usize = 128;
+pub const CRYPTO_CIPHERTEXTBYTES: usize = 96;
```

Comment:
The `CryptoCiphertextBytesTypenum` type must also be updated for all variants. The code can't currently work with the `kem` feature enabled.

Reply:
thx for your review @faern, yes these types would have to be changed, I also noticed while enabling the `kem` feature in cargo toml, that we don't catch a not-working `kem` feature yet while running `cargo test`, so I'm in favour of writing a test for it, I can do this in a seperate PR

Reply:
I've updated `CryptoCiphertextBytesTypenum` accordingly. Should I resolve this conversation?

Reply:
thank you for updating the PR, please await the conversation resolve by the reviewer

Reply:
> also would you happen to know how to use debug statements like `dbg!` in `lib.rs` ?

What do you mean? What is it that does/doesn't happen if you just add it? Do you mean to debug while running tests? You need to pass `--nocapture` to `cargo test` for it to not hide stderr (`cargo test -- --nocapture`). If you want to keep debug statements in production we should use `log::debug!` and not `dbg!`.

Reply:
The `kem` tests really need fixing! It's a bit scary that they passed :see_no_evil:

Reply:
> > also would you happen to know how to use debug statements like `dbg!` in `lib.rs` ?
> 
> What do you mean? What is it that does/doesn't happen if you just add it? Do you mean to debug while running tests? You need to pass `--nocapture` to `cargo test` for it to not hide stderr (`cargo test -- --nocapture`). If you want to keep debug statements in production we should use `log::debug!` and not `dbg!`.

I'll try to be more precise: I wanted to debug the `recip_pubkey` variable in the `try_encap` function in lib.rs, however when inserting for instance `dbg!(recip_pubkey);` I get: `error: cannot find macro `dbg` in this scope`
Exactly, I meant to debug during running this specific test :)

so we would still need to prove that the `kem` feature runs correctly ?

Reply:
Ah, yes, the `dbg!` macro is in `std`. This crate is a `#![no_std]` crate. Remove that line for temporarily using it. Like here: https://github.com/mullvad/classic-mceliece-rust/commit/82fbdfc45ba0b3b0f84248fde19144d1e13e0edd